packages/react-native/src/primitives/Heading/types.ts

```diff
@@ -0,0 +1,15 @@
+import { StyleProp, TextProps, TextStyle } from 'react-native';
+
+export type HeadingLevel = 1 | 2 | 3 | 4 | 5 | 6;
```

Comment:
Not seeing this imported elsewhere in the PR so i think we can remove the `export` since we are only using within this file. With it being scoped to just this file, what do you think of renaming to just `Level`?